src/commands/clean.ts

```diff
@@ -0,0 +1,23 @@
+import { workspace } from 'vscode';
+import { ExtensionContainer } from '../container';
+import { cleanAppArguments } from '../utils';
+import {  handleInteractionError, InteractionError } from './common';
+
+import { CleanAppOptions } from '../types/cli';
+
+export async function cleanApplication () {
+	try {
+		const logLevel = ExtensionContainer.config.general.logLevel;
+		const projectDir = workspace.rootPath;
```

Comment:
For a module project, `projectDir` needs to point at a platform subfolder i.e. `~/Documents/Appcelerator_Studio_Workspace/mymodule/android`. If we're in a module directory do we want to throw a select dialog for the platform(s?) to clean?